manifests/07_deployment.yaml

```diff
@@ -53,6 +56,12 @@ spec:
         configMap:
           defaultMode: 440
           name: authentication-operator-config
+      - name: trusted-ca-bundle
+        configMap:
+          name: trusted-ca
+          items:
+          - key: ca-bundle.crt
+            path: tls-ca-bundle.pem
```

Comment:
Why do we want a different path instead of using `ca-bundle.crt`?

Reply:
@enj this was the default chosen for all operators as referenced in the [bootstrap workflow](https://docs.google.com/document/d/1y0t0yEOSnKc4abxsjxEQjrFa1AP8iHcGyxlBpqGLO08/edit?usp=sharing) doc.

Reply:
Sure but there is no need to mount that config map at `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem` in the authentication operator.  It is all custom Go code / the Go std lib does not look at that path for system roots.

Reply:
I think Go stdlib actually might look at those files (see `crypto/x509/root_linux.go`) but loading from a specific file sounds like a better idea than relying on this specific defaulting logic, for now at least, since we can easily control all the connections we make outside. It's something to bear in mind, though.